app/_src/networking/transparent-proxy/introduction.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Transparent Proxy
+content_type: how-to
+---
+
+{% assign docs = "/docs/" | append: page.version %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign tproxy = site.data.tproxy %}
+
+{% capture tproxy-config-reference-info %}
+For a full list of transparent proxy settings with examples and configuration options, see the [Transparent Proxy Configuration Reference]({{ docs }}/reference/transparent-proxy-configuration/).
+{% endcapture %}
+
+A transparent proxy is a server that intercepts network traffic going to and from a service without requiring any changes to the application code. In {{ Kuma }}, it captures this traffic and routes it to the [data plane proxy]({{ docs }}/production/dp-config/dpp/#data-plane-proxy), allowing [Mesh policies]({{ docs }}/policies/introduction/#policies) to be applied.
+
+{{ Kuma }} uses [iptables](https://linux.die.net/man/8/iptables) and also has experimental support for [`eBPF`](#transparent-proxy-with-ebpf-experimental) to make this possible.
```

Comment:
```suggestion
{{ Kuma }} uses [iptables](https://linux.die.net/man/8/iptables) to make this possible. Moreover, {{ Kuma }} offers experimental support for [eBPF](https://ebpf.io/) 
```

I don't like mixing special formatting with links. When I was reading this doc I didn't know it was link until I moved my mouse over it by accident. Also I think we could link to eBPF here as we link to iptables

Reply:
addressed